addons/account/models/account.py

```diff
@@ -462,6 +462,23 @@ def action_open_reconcile(self):
             'context': action_context,
         }
 
+    def action_open_account(self):
+        self.ensure_one()
+        return {
+            'name': self.display_name,
+            'type': 'ir.actions.act_window',
+            'view_type': 'form',
+            'view_mode': 'form',
+            'res_model': 'account.account',
+            'res_id': self.id,
+            'target': 'new',
+        }
+
+    def duplicate_accounts(self):
+        account_ids = self.browse(self.env.context['active_ids'])
+        for account_id in account_ids:
```

Comment:
```suggestion
        for account in self.browse(self.env.context['active_ids']):
```
This makes more sense